### PR TITLE
Adding note for initial commit requirement during quickstart

### PR DESCRIPTION
**Pull request by BryanFauble**
**Problem:**

1. A `quickstart.md` exists in 2 locations making maintenance of the file an extra chore.

**Solution:**

1. Updating the root `quickstart.md` to redirect in the browser to the expected location.

**Testing:**

1. Verified on my local machine when I use a `mkdocs serve` and load it into firefox my browser correctly redirected.
2. I also verified without the auto redirect that clicking the text works as expected.

**-------------- Original issue that was not a kubefox bug that needed to be addressed: --------------**

**Problem:**
- During the quickstart docs it has you do:
```
mkdir kubefox-quickstart && \
  cd kubefox-quickstart && \
  export FOX_INFO=true && \
  fox init --quickstart
```
(The same as the screenshot below, but I did it over multiple steps as I was debugging the issue in the first place)

- 2 steps after this init it is followed by:
```
fox publish --wait 5m
```

However - This fails as this is a newly create git repo with no existing commits:
image

**Solution:**
- I was able to just add a commit with all of the initial example files and `fox publish` as expected

**Testing:**
- I verified on an Arch linux distro that I was able to follow the full quickstart guide from start to end
image

### CODE REVIEW

**Comment by BryanFauble** [comment]
@xadhatter There are 2 quickstart.md files. The one at `docs/quickstart.md` doesn't appear to be referenced anywhere; They are identical. Do they both need to exist, or can I removed `docs/quickstart.md`?

**Comment by xadhatter** [comment]
> @xadhatter There are 2 quickstart.md files. The one at `docs/quickstart.md` doesn't appear to be referenced anywhere; They are identical. Do they both need to exist, or can I removed `docs/quickstart.md`?

We had sent out material with links to the Quickstart in the root docs directory so we temporarily left it. It'd be better to leave the page but have it redirect to the Quickstart under `Getting Started`.

**Comment by xadhatter** [comment]
> However - This fails as this is a newly create git repo with no existing commits:



This appears to be a bug in the Fox CLI. Can you send the output of `git status` after you perform the `fox init` command?

Thanks so much for taking the time to test and for the PR.

**Comment by BryanFauble** [comment]
> > However - This fails as this is a newly create git repo with no existing commits:
> 
> This appears to be a bug in the Fox CLI as it should be performing the initial commit. Can you send the output of `fox --verbose init` followed by `git status`?
> 
> Thanks so much for taking the time to test and for the PR.

I went through and verified that it was a missing `~/.gitconfig`. When it is missing and an error occurs we do get back:
`error   😖 Error committing changes: author field is required`

I verified that after adding `~/.gitconfig` this worked as expected.

File: docs/quickstart.md [hunk 1]
```diff
@@ -1,1064 +1,9 @@
...
-by manually providing context. KubeFox needs two pieces of information to route
-an event, the AppDeployment to use and the Virtual Environments to inject. These
-can be passed as headers or query parameters for HTTP requests.
-
-```{ .shell .copy }
-curl "http://localhost:8080/qa/hello?kf-dep=hello-world-main&kf-ve=qa"
-```
-
-??? example "Output"
+<meta http-equiv="refresh" content="0; url=getting_started/quickstart.html" />
```
Comment (BryanFauble) [inline docs/quickstart.md:1]: @xadhatter This appears to work as expected when I deploy the mkdocs to my local machine - In lieu of the automatic redirect working I added in a manual button they can click to go to the new page.

**Review by xadhatter — CHANGES_REQUESTED** [review]
Nice fix. Thanks.

**Review by xadhatter — APPROVED** [review]

**Pull request merged**